### PR TITLE
Node example

### PR DESCRIPTION
**Pull request by Patrick-Willis**
Added a little example based on what I through together for feature testing.

### CODE REVIEW

**Comment by Patrick-Willis** [comment]
Thanks @matt, @JMontagu is going to have a look at making it more idiomatic I think

File: Node/example.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      'X-ContentName': [POSITIONS FILE NAME],
+      // use this line to define the file type. e.g. 'text/xml', 'application/zip'
+      'Content-Type': [POSITIONS FILE TYPE],
+    },
+    // connect to the HTTPS endpoint with the API credentials you have been provided
+    uri: 'https://[ALIAS]-api.fundapps.co/v1/expost/check',
+    method: 'POST',
+    body: data,
+    auth: {
+      user: [USERNAME],
```
Comment (JMontagu) [inline Node/example.js]: `
user: process.env.USERNAME,
pass: process.env.PASSWORD,
`

File: Node/example.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    },
+  };
+}
+
+function sendFile(data) {
+  request(requestOptions(data), logResult);
+}
+
+fs.readFile([POSITIONS FILE], (error, data) => {
+  if (error == null) {
```
Comment (JMontagu) [inline Node/example.js]:
```
if (!error) {
  console.error(error);
}

sendFile(data)
```

File: Node/example.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+// Example using request (https://github.com/request/request)
+
+const request = require('request');
+const fs = require('fs');
+
+// if response comes back with a 2xx status, then file was received successfully
+function logResult(error, response, body) {
+  console.log(error);
```
Comment (JMontagu) [inline Node/example.js]: `console.error`

**Pull request merged**